apis/slo/v1alpha1/nodeslo_types.go

```diff
@@ -23,6 +23,107 @@ import (
 // EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
 // NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
 
+// MemoryQOS enables memory qos features.
+type MemoryQOS struct {
```

Comment:
QoS is a unique shorthand, it is recommended to follow

Reply:
done